.eslintrc.js

```diff
@@ -195,11 +195,16 @@
                 }],
                 "@typescript-eslint/no-explicit-any": 0,
                 "@typescript-eslint/adjacent-overload-signatures": "error",
-                "@typescript-eslint/ban-ts-ignore": "error",
-                "@typescript-eslint/ban-types": "error",
```

Comment:
ban-types was changed entirely from the originally rule, most of our code was failing with it so it seemed prudent to turn it off.